contrib/babelfishpg_common/src/varbinary.c

```diff
@@ -1503,6 +1506,67 @@ varbinary_leq(PG_FUNCTION_ARGS)
 	PG_RETURN_BOOL(result);
 }
 
+Datum
+int4varbinary_div(PG_FUNCTION_ARGS)
+{
+       int32      dividend = PG_GETARG_INT32(0);
+       bytea      *varbinary_divisor = PG_GETARG_BYTEA_PP(1);
```

Comment:
can it be NULL in any case, handle to be on safe side

Reply:
No it cannot be NULL , it can be zero which is being handled.